vendor/bat-native-ledger/src/bat/ledger/internal/database/database_unblinded_token.cc

```diff
@@ -442,7 +442,8 @@ void DatabaseUnblindedToken::GetRecordsByTriggerIds(
       "SELECT ut.token_id, ut.token_value, ut.public_key, ut.value, "
       "ut.creds_id, ut.expires_at FROM %s as ut "
       "LEFT JOIN creds_batch as cb ON cb.creds_id = ut.creds_id "
-      "WHERE cb.trigger_id IN (%s) OR ut.creds_id IS NULL",
+      "WHERE ut.redeemed_at = 0 AND "
```

Comment:
this is an actual change as we should only return tokens that has `redeemed_at` equal 0